### PR TITLE
Release/0.4.1

### PR DESCRIPTION
**Pull request by rzats**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Version 0.4.1 (2019-02-15)
```
Comment (chuwy) [inline CHANGELOG]: Date

File: core/src/main/scala/com.snowplowanalytics.snowflake.core/Config.scala [hunk 2]
```diff
@@ -56,8 +58,22 @@ object Config {
   case object SetupCommand extends Command
   case object MigrateCommand extends Command
 
-  case class RawCliLoader(command: String, loaderConfig: String, resolver: String, loaderVersion: String, dryRun: Boolean, base64: Boolean)
-  case class CliLoaderConfiguration(command: Command, loaderConfig: Config, loaderVersion: String, dryRun: Boolean)
+  sealed trait SetupSteps extends EnumEntry
+
+  object SetupSteps extends Enum[SetupSteps] {
+    val values = findValues
```
Comment (chuwy) [inline core/src/main/scala/com.snowplowanalytics.snowflake.core/Config.scala:64]: Is it necessary?

File: core/src/main/scala/com.snowplowanalytics.snowflake.core/Config.scala [hunk 3]
```diff
@@ -159,13 +175,18 @@ object Config {
...
   private val loaderCliParser = new scopt.OptionParser[RawCliLoader](ProjectMetadata.name + "-" + ProjectMetadata.version + ".jar") {
     head(ProjectMetadata.name, ProjectMetadata.version)
 
     cmd("setup")
       .action((_, c) => c.copy(command = "setup"))
       .text("Perform initialization instead of loading")
+      .children(
+        opt[Seq[SetupSteps]]("skip")
+          .action((x, c) => c.copy(skip = x.toSet))
+          .text("A comma-separated list of setup steps to skip. Possible values: schema, table, warehouse, fileformat, stage")
```
Comment (chuwy) [inline core/src/main/scala/com.snowplowanalytics.snowflake.core/Config.scala]: Use "single source of truth" principle here.

**Review by chuwy — APPROVED** [review]
Just couple of nits. 👍 otherwise.

File: core/src/main/scala/com.snowplowanalytics.snowflake.core/Config.scala (hunk 2 above)
Comment (rzats) [reply core/src/main/scala/com.snowplowanalytics.snowflake.core/Config.scala:64]: Yes, it's what the enumeration library uses to make `withName*` methods work - in fact it's required to implement the `Enum` class.

**Pull request merged**